### PR TITLE
Menu Enhancements to support Player 'AlwaysOnTop' and Show/Hide of Menu

### PR DESCRIPTION
**Pull request by f3lin3**
These are a few quick enhancements to the Player's MenuBar for your consideration:

1. Support the Player staying on top (which is particularly useful if in windowed mode);
2. Support explicit hiding and showing of the MenuBar.

### CODE REVIEW

**Comment by ififfy** [comment]
Adding fun new menu items like this is pretty uncontroversial. :-)

File: src/renderer/components/player/Player.tsx [hunk 1]
```diff
@@ -17,6 +17,8 @@ const keyMap = {
   historyForward: ['Forward in time', 'right'],
   navigateBack: ['Go back to scene details', 'backspace'],
   toggleFullscreen: ['Toggle fullscreen', 'CommandOrControl+F'],
+  alwaysOnTop: ['Toggle On Top', 'CommandOrControl+T'],
+  toggleMenuBarDisplay: ['Show/Hide Menu', 'CommandOrControl+M'],
```
Comment (ififfy) [inline src/renderer/components/player/Player.tsx]: Please choose another shortcut. Command+M on Mac is Minimize and can't be overridden.

Comment (f3lin3) [reply src/renderer/components/player/Player.tsx]: I changed it to ComandOrControl + Shift + 6  - and have checked it in - do I need to make a new PR?

Comment (GoddessServant0) [reply src/renderer/components/player/Player.tsx]: If you commit it to your origin/master then it should show up here 🤔 (Make sure it's pushed)

Comment (GoddessServant0) [reply src/renderer/components/player/Player.tsx]: No need to make a new PR 👍

**Pull request merged**